modules/Mut/Styles/CheckBoxStyle.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.5
+import QtQuick.Controls.Styles 1.3
+import QtQuick.Controls 1.3 
+import Mut 0.1
+import Mut.Themes 0.1
+
+CheckBoxStyle {
+    id: root 
+
+    spacing: 0
+    property CheckBoxPalette palette: Theme.p.checkBox
+    property color backgroundColor: {
+        control.checked ?
+            control.enabled ? palette.backgroundSelected : palette.backgroundSelectedDisabled
+        : palette.backgroundUnselected
+    }
+
+    property color borderColor: {
+        control.checked ? palette.backgroundUnselected :
+        control.enabled ? palette.borderEnabled : palette.borderDisabled
+    }
+
+    label: Rectangle {
+        id: labelRectangle
+        height: Units.dp(48)
+
+        Text {
+            id: text
+            anchors.verticalCenter: parent.verticalCenter
+            text: control.text
+        
+            MouseArea {
+                anchors.fill: parent
+                onClicked: control.checked = !control.checked
+            }
+        }
+    }
+    
+
+    indicator: Item {
+        id: parentRect
+            
+        implicitWidth: Units.dp(48)
+        implicitHeight: implicitWidth
```

Comment:
this creates an unidirectional binding so that every time ``implicitWidth`` changes, also ``implicitHeight`` will change. Because you will never use such bindings, it's better to prevent these connections. I will write simply:

```
implicitWidth: Units.dp(48)
implicitHeight: Units.dp(48)
```